lib/glueby/contract/tx_builder.rb

```diff
@@ -8,7 +8,7 @@ module TxBuilder
       # @raise [InsufficientFunds] if result of listunspent is not enough to pay the specified amount
       def collect_outputs(results, amount)
         results.inject([0, []]) do |sum, output|
-          new_sum = sum[0] + (output['amount'] * 100_000_000)
+          new_sum = sum[0] + (output['amount'].to_i * 100_000_000)
```

Comment:
The values returned by listunspent include values after the decimal point, such as `"0.41547730"`. 
In that case, this code output `0` value.

And please implement unit test.

Reply:
OK, Thank you review.
I will fix it, and run unit tests.